### DISCUSSION TITLE
Support for international date conventions

### DISCUSSION THREAD

**Discussion by ritwikvd** [community/community#16693] [Other Feature Feedback, Questions, & Ideas]
I understand that GitHub is an American company but if it's catering to a global developer audience it should mould its product accordingly.

International conventions, regarding the use of standard date notation `yyyy/mm/dd` or `dd/mm/yyyy`, should be the norm.

Alternatively, github could implement localisation so that users view everything in whatever format is in prevalent use in their region.

`5/15/2022` is simply befuddling for people not acquainted with it, and is largely invalid outside of NA. It's the same case for the long format `dd Mon yyyy`.

**Comment by ExcuseLme** [comment]
I try to search for "date format" in github/feedback/discussions, and got a lot of search results, but there was no official response to any of them, it's frustrating
